repository-module/src/main/java/ru/mail/ilya6697089/impl/LinesCollectionRepositoryImpl.java

```diff
@@ -0,0 +1,45 @@
+package ru.mail.ilya6697089.impl;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.springframework.stereotype.Repository;
+import ru.mail.ilya6697089.LinesCollectionRepository;
+
+@Repository
+public class LinesCollectionRepositoryImpl implements LinesCollectionRepository {
+
+    private static final Logger logger = LogManager.getLogger(MethodHandles.lookup().lookupClass());
+
+    @Override
+    public List<String> getLinesCollection(String fileName) {
+        try {
+            File file = new File(fileName);
+            FileReader fr = new FileReader(file);
+            BufferedReader reader = new BufferedReader(fr);
```

Comment:
Should use try-with-resources section to close the stream. bad practice - potential memory leak